sdk/src/contexts/solana/context.ts

```diff
@@ -1073,37 +1098,49 @@ export class SolanaContext<
           }
         }
       }
-      transferIx = await createTransferNativeTokensWithRelayInstruction(
-        this.connection,
-        relayer,
-        senderAddress,
-        token_bridge,
-        core,
-        mint,
-        BigInt(amount),
-        BigInt(toNativeToken),
-        formattedRecipient,
-        recipientChainId,
-        nonce,
-        wrapToken,
+
+      transaction.add(
+        ...(await this.determineComputeBudget(writableAddresses)),
+      );
+      transaction.add(
+        await createTransferNativeTokensWithRelayInstruction(
```

Comment:
could do this and the above concurrently?

Reply:
valid suggestion but IMO not worth the code complexity